.gitbook/guide/interaction-with-the-network-cli.md

```diff
@@ -279,21 +279,50 @@ panaceacli tx staking create-validator \
   For details about various key types, please see this [guide](interaction-with-the-network-cli.md#keys).
 - `moniker`: A validator nickname that will be displayed publicly
 - `commission-rate`: An initial commission rate on block rewards and fees charged to delegators
-- `commission-max-rate`: A maximum commission rate which this validator can charge. This can be changed after the
+- `commission-max-rate`: A maximum commission rate which this validator can charge. This cannot be changed after the
   `create-validator` transaction is processed.
-- `commission-max-change-rate`: A maximum daily increase of the validator commission. This can be changed after the
+- `commission-max-change-rate`: A maximum daily increase of the validator commission. This cannot be changed after the
   `create-validator` transaction is processed. This is used to measure % point change over the `commision-rate`.
   E.g. 1% to 2% is a 100% rate increase, but only 1% point.
-- `min-self-delegation`: A strictly positive integer that represents the minimum amount of self-delegated voting power
-  your validator must always have. A `min-self-delegation` of `1` means your validator will never have a self-delegation
-  lower than `1med` (`1000000umed`).
+- `min-self-delegation`: A strictly positive integer in `umed` that represents the minimum amount of self-delegated voting power
+  your validator must always have. If the validator's self-delegated stake falls below this limit, their entire staking pool will unbond.
```

Comment:
The unit of `min-self-delegation` is still unclear. I guess it's a bug of Cosmos SDK: https://github.com/cosmos/cosmos-sdk/issues/9386.

Reply:
Well. It was just confirmed by Cosmos team that the unit is `umed`. https://github.com/cosmos/cosmos-sdk/issues/9386#issuecomment-847134820